tools/rpkg/src/relational.cpp

```diff
@@ -198,6 +225,70 @@ external_pointer<T> make_external_prot(const string &rclass, SEXP prot, ARGS &&.
 	return make_external_prot<RelationWrapper>("duckdb_relation", prot, res);
 }
 
+static WindowBoundary StringToWindowBoundary(string &window_boundary) {
+	if (window_boundary == "unbounded_preceding") {
```

Comment:
can't tell if this is already implemented but: can you double-check those strings-as-enums are case-insensitive, here and also for the joins etc in the R layer?

Reply:
`src/common/serializer/enum_serializer.cpp` has them in upper case.

Reply:
I've wrapped the boundary start and boundary end in a `tolower()` function.
https://github.com/duckdb/duckdb/pull/6757/files#diff-18937569a862b0f2ed2261cdf43488c1ccc5f115848477098bb232d376ea2359R180

Also have tests for this in test_relational.cpp